src/command.rs

```diff
@@ -38,6 +64,12 @@ pub fn run_wasm_pack(command: Command) -> result::Result<(), Error> {
         Command::Init { path, scope } => init(path, scope),
         Command::Pack { path } => pack(path),
         Command::Publish { path } => publish(path),
+        Command::Adduser {
+            registry,
+            scope,
+            always_auth,
+            auth_type,
+        } => adduser(registry, scope, always_auth, auth_type),
```

Comment:
I feel like this should be the exception and just have npm::add_user be called directly. Though I also get having consistency. @ashleygwilliams thoughts?